src/App.config

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <configSections>
+    <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog" />
+  </configSections>
+  <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <targets>
+      <target name="ASFile" xsi:type="AsyncWrapper" overflowAction="Block" queueLimit="10000" batchSize="1000" 
+              timeToSleepBetweenBatches="100">
+        <target xsi:type="File" name="file" keepFileOpen="true" openFileCacheTimeout="5" archiveNumbering="Date" 
+                archiveEvery="Day" archiveDateFormat="yyyyMMdd" encoding="utf-8" enableArchiveFileCompression="true"
+                fileName="${environment:variable=APPDATA:whenEmpty=${environment:variable=HOME}}\q2g\q2g-con-ser-qvx\log.log"
```

Comment:
@mossbeard please change the name q2g -> senseexcel q2g-con-ser-qvx to the real name